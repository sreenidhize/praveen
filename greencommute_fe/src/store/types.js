export default {
  ADD_YOUR_LOCATION: "ADD_YOUR_LOCATION",
  ADD_JOB_LOCATIONS: "ADD_JOB_LOCATIONS",
  ADD_SKILLS: "ADD_SKILLS",
  ADD_JOBS: "ADD_JOBS",
  SET_SAVED_JOBS: "SET_SAVED_JOBS",
  ADD_SAVED_JOB: "ADD_SAVED_JOB",
  REMOVE_SAVED_JOB: "REMOVE_SAVED_JOB",
  APPLY_FILTER: "APPLY_FILTER",
  REMOVE_FILTER: "REMOVE_FILTER",
  ADD_FILTERED_JOBS: "ADD_FILTERED_JOBS",
  REMOVE_FILTERED_JOBS: "REMOVE_FILTERED_JOBS",
  REMOVE_EACH_FILTER: "REMOVE_EACH_FILTER",
};
