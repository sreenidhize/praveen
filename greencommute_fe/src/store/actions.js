import types from "./types";

export const addYourLocation = (payload) => ({
  payload: payload,
  type: types.ADD_YOUR_LOCATION,
});

export const addJobLocations = (payload) => ({
  payload: payload,
  type: types.ADD_JOB_LOCATIONS,
});

export const addSkills = (payload) => ({
  payload: payload,
  type: types.ADD_SKILLS,
});

export const addJobs = (payload) => ({
  payload: payload,
  type: types.ADD_JOBS,
});

export const addAdvancedFilterJobs = (payload) => ({
  payload: payload,
  type: types.ADD_FILTERED_JOBS,
});

export const removeEachFilter = (payload) => ({
  payload,
  type: types.REMOVE_EACH_FILTER,
});

export const removeAdvancedFilterJobs = (payload) => ({
  payload: payload,
  type: types.REMOVE_FILTERED_JOBS,
});

export const addSavedJob = (payload) => ({
  payload,
  type: types.ADD_SAVED_JOB,
});

export const removeSavedJob = (payload) => ({
  payload,
  type: types.REMOVE_SAVED_JOB,
});

export const setSavedJobs = (payload) => ({
  payload,
  type: types.SET_SAVED_JOBS,
});

export const applyAdvancedFilters = (payload) => ({
  payload: payload,
  type: types.APPLY_FILTER,
});

export const removeAdvancedFilters = () => ({
  type: types.REMOVE_FILTER,
});

export default {
  addYourLocation,
  addJobLocations,
  addSkills,
  addJobs,
  addAdvancedFilterJobs,
  removeAdvancedFilterJobs,
  applyAdvancedFilters,
  removeAdvancedFilters,
  removeEachFilter,
  addSavedJob,
  removeSavedJob,
  setSavedJobs,
};
