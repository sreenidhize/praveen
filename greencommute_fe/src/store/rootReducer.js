//import reducers here
import { combineReducers } from "redux";
import onBoardingFilter from "./reducers/onBoardingFilter";
import jobReducer from "./reducers/jobs";
import advancedFilters from "./reducers/advancedFilter";

const rootReducer = combineReducers({
  //add reduceres here
  onBoardingFilter,
  jobReducer,
  advancedFilters,
});

export default rootReducer;
