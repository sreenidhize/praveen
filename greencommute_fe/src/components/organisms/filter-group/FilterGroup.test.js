import React from "react";
import FilterGroup from "./FilterGroup";
import { render } from "@testing-library/react";
import { Provider } from "react-redux";
import store from "../../../store/configStore";

describe("component renders correctly", () => {
  test("should match snapshot", () => {
    const { asFragment, debug } = render(
      <Provider store={store}>
        <FilterGroup />
      </Provider>
    );
    expect(asFragment()).toMatchSnapshot();
  });

  test("block should not be empty", () => {
    const { debug, container } = render(
      <Provider store={store}>
        <FilterGroup />
      </Provider>
    );

    expect(container.getElementsByTagName("div").length).toBe(18);
  });
});
