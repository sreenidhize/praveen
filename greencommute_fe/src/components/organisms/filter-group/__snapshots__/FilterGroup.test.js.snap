// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component renders correctly should match snapshot 1`] = `
<DocumentFragment>
  <div
    class="MuiGrid-root MuiGrid-container"
  >
    <div
      class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-justify-xs-space-evenly"
    >
      <div
        class="MuiGrid-root MuiGrid-item"
      >
        <div
          class="MuiFormGroup-root"
        >
          <p
            class="MuiTypography-root makeStyles-textMargin-4 MuiTypography-body1"
          >
            Distance
          </p>
          <label
            class="MuiFormControlLabel-root makeStyles-formLabel-2"
          >
            <span
              aria-disabled="false"
              class="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-5 MuiCheckbox-root MuiCheckbox-colorPrimary MuiIconButton-colorPrimary"
            >
              <span
                class="MuiIconButton-label"
              >
                <input
                  class="PrivateSwitchBase-input-8"
                  data-indeterminate="false"
                  name="0 - 10 kms"
                  type="checkbox"
                  value=""
                />
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
                  />
                </svg>
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </span>
            <span
              class="MuiTypography-root MuiFormControlLabel-label MuiTypography-body1"
            >
              <h4
                class="MuiTypography-root makeStyles-label-1 MuiTypography-h4"
              >
                0 - 10 kms
              </h4>
            </span>
          </label>
          <label
            class="MuiFormControlLabel-root makeStyles-formLabel-2"
          >
            <span
              aria-disabled="false"
              class="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-5 MuiCheckbox-root MuiCheckbox-colorPrimary MuiIconButton-colorPrimary"
            >
              <span
                class="MuiIconButton-label"
              >
                <input
                  class="PrivateSwitchBase-input-8"
                  data-indeterminate="false"
                  name="11 - 20 kms"
                  type="checkbox"
                  value=""
                />
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
                  />
                </svg>
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </span>
            <span
              class="MuiTypography-root MuiFormControlLabel-label MuiTypography-body1"
            >
              <h4
                class="MuiTypography-root makeStyles-label-1 MuiTypography-h4"
              >
                11 - 20 kms
              </h4>
            </span>
          </label>
          <label
            class="MuiFormControlLabel-root makeStyles-formLabel-2"
          >
            <span
              aria-disabled="false"
              class="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-5 MuiCheckbox-root MuiCheckbox-colorPrimary MuiIconButton-colorPrimary"
            >
              <span
                class="MuiIconButton-label"
              >
                <input
                  class="PrivateSwitchBase-input-8"
                  data-indeterminate="false"
                  name="21 - 30 kms"
                  type="checkbox"
                  value=""
                />
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
                  />
                </svg>
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </span>
            <span
              class="MuiTypography-root MuiFormControlLabel-label MuiTypography-body1"
            >
              <h4
                class="MuiTypography-root makeStyles-label-1 MuiTypography-h4"
              >
                21 - 30 kms
              </h4>
            </span>
          </label>
          <label
            class="MuiFormControlLabel-root makeStyles-formLabel-2"
          >
            <span
              aria-disabled="false"
              class="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-5 MuiCheckbox-root MuiCheckbox-colorPrimary MuiIconButton-colorPrimary"
            >
              <span
                class="MuiIconButton-label"
              >
                <input
                  class="PrivateSwitchBase-input-8"
                  data-indeterminate="false"
                  name="31 - 40 kms"
                  type="checkbox"
                  value=""
                />
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
                  />
                </svg>
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </span>
            <span
              class="MuiTypography-root MuiFormControlLabel-label MuiTypography-body1"
            >
              <h4
                class="MuiTypography-root makeStyles-label-1 MuiTypography-h4"
              >
                31 - 40 kms
              </h4>
            </span>
          </label>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item"
      >
        <div
          class="MuiFormGroup-root"
        >
          <p
            class="MuiTypography-root makeStyles-textMargin-4 MuiTypography-body1"
          >
            Date Posted
          </p>
          <label
            class="MuiFormControlLabel-root makeStyles-formLabel-2"
          >
            <span
              aria-disabled="false"
              class="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-5 MuiCheckbox-root MuiCheckbox-colorPrimary MuiIconButton-colorPrimary"
            >
              <span
                class="MuiIconButton-label"
              >
                <input
                  class="PrivateSwitchBase-input-8"
                  data-indeterminate="false"
                  name="Past 24 Hours"
                  type="checkbox"
                  value=""
                />
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
                  />
                </svg>
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </span>
            <span
              class="MuiTypography-root MuiFormControlLabel-label MuiTypography-body1"
            >
              <h4
                class="MuiTypography-root makeStyles-label-1 MuiTypography-h4"
              >
                Past 24 Hours
              </h4>
            </span>
          </label>
          <label
            class="MuiFormControlLabel-root makeStyles-formLabel-2"
          >
            <span
              aria-disabled="false"
              class="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-5 MuiCheckbox-root MuiCheckbox-colorPrimary MuiIconButton-colorPrimary"
            >
              <span
                class="MuiIconButton-label"
              >
                <input
                  class="PrivateSwitchBase-input-8"
                  data-indeterminate="false"
                  name="Past Week"
                  type="checkbox"
                  value=""
                />
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
                  />
                </svg>
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </span>
            <span
              class="MuiTypography-root MuiFormControlLabel-label MuiTypography-body1"
            >
              <h4
                class="MuiTypography-root makeStyles-label-1 MuiTypography-h4"
              >
                Past Week
              </h4>
            </span>
          </label>
          <label
            class="MuiFormControlLabel-root makeStyles-formLabel-2"
          >
            <span
              aria-disabled="false"
              class="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-5 MuiCheckbox-root MuiCheckbox-colorPrimary MuiIconButton-colorPrimary"
            >
              <span
                class="MuiIconButton-label"
              >
                <input
                  class="PrivateSwitchBase-input-8"
                  data-indeterminate="false"
                  name="Past month"
                  type="checkbox"
                  value=""
                />
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
                  />
                </svg>
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </span>
            <span
              class="MuiTypography-root MuiFormControlLabel-label MuiTypography-body1"
            >
              <h4
                class="MuiTypography-root makeStyles-label-1 MuiTypography-h4"
              >
                Past month
              </h4>
            </span>
          </label>
          <label
            class="MuiFormControlLabel-root makeStyles-formLabel-2"
          >
            <span
              aria-disabled="false"
              class="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-5 MuiCheckbox-root MuiCheckbox-colorPrimary MuiIconButton-colorPrimary"
            >
              <span
                class="MuiIconButton-label"
              >
                <input
                  class="PrivateSwitchBase-input-8"
                  data-indeterminate="false"
                  name="Anytime"
                  type="checkbox"
                  value=""
                />
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
                  />
                </svg>
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </span>
            <span
              class="MuiTypography-root MuiFormControlLabel-label MuiTypography-body1"
            >
              <h4
                class="MuiTypography-root makeStyles-label-1 MuiTypography-h4"
              >
                Anytime
              </h4>
            </span>
          </label>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item"
      >
        <div
          class="MuiFormGroup-root"
        >
          <p
            class="MuiTypography-root makeStyles-textMargin-4 MuiTypography-body1"
          >
            Green Commute
          </p>
          <label
            class="MuiFormControlLabel-root makeStyles-formLabel-2"
          >
            <span
              aria-disabled="false"
              class="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-5 MuiCheckbox-root MuiCheckbox-colorPrimary MuiIconButton-colorPrimary"
            >
              <span
                class="MuiIconButton-label"
              >
                <input
                  class="PrivateSwitchBase-input-8"
                  data-indeterminate="false"
                  name="Yes"
                  type="checkbox"
                  value=""
                />
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
                  />
                </svg>
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </span>
            <span
              class="MuiTypography-root MuiFormControlLabel-label MuiTypography-body1"
            >
              <h4
                class="MuiTypography-root makeStyles-label-1 MuiTypography-h4"
              >
                Yes
              </h4>
            </span>
          </label>
        </div>
      </div>
    </div>
    <div
      class="MuiGrid-root makeStyles-gridMargin-3 MuiGrid-container MuiGrid-item MuiGrid-justify-xs-space-evenly"
    >
      <div
        class="MuiGrid-root MuiGrid-item"
      >
        <div
          class="MuiFormGroup-root"
        >
          <p
            class="MuiTypography-root makeStyles-textMargin-4 MuiTypography-body1"
          >
            Job Type
          </p>
          <label
            class="MuiFormControlLabel-root makeStyles-formLabel-2"
            variant="secondary"
          >
            <span
              aria-disabled="false"
              class="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-5 MuiCheckbox-root MuiCheckbox-colorPrimary MuiIconButton-colorPrimary"
            >
              <span
                class="MuiIconButton-label"
              >
                <input
                  class="PrivateSwitchBase-input-8"
                  data-indeterminate="false"
                  name="Full-Time"
                  type="checkbox"
                  value=""
                />
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
                  />
                </svg>
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </span>
            <span
              class="MuiTypography-root MuiFormControlLabel-label MuiTypography-body1"
            >
              <h4
                class="MuiTypography-root makeStyles-label-1 MuiTypography-h4"
              >
                Full-Time
              </h4>
            </span>
          </label>
          <label
            class="MuiFormControlLabel-root makeStyles-formLabel-2"
            variant="secondary"
          >
            <span
              aria-disabled="false"
              class="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-5 MuiCheckbox-root MuiCheckbox-colorPrimary MuiIconButton-colorPrimary"
            >
              <span
                class="MuiIconButton-label"
              >
                <input
                  class="PrivateSwitchBase-input-8"
                  data-indeterminate="false"
                  name="Internship"
                  type="checkbox"
                  value=""
                />
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
                  />
                </svg>
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </span>
            <span
              class="MuiTypography-root MuiFormControlLabel-label MuiTypography-body1"
            >
              <h4
                class="MuiTypography-root makeStyles-label-1 MuiTypography-h4"
              >
                Internship
              </h4>
            </span>
          </label>
          <label
            class="MuiFormControlLabel-root makeStyles-formLabel-2"
            variant="secondary"
          >
            <span
              aria-disabled="false"
              class="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-5 MuiCheckbox-root MuiCheckbox-colorPrimary MuiIconButton-colorPrimary"
            >
              <span
                class="MuiIconButton-label"
              >
                <input
                  class="PrivateSwitchBase-input-8"
                  data-indeterminate="false"
                  name="Contract"
                  type="checkbox"
                  value=""
                />
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
                  />
                </svg>
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </span>
            <span
              class="MuiTypography-root MuiFormControlLabel-label MuiTypography-body1"
            >
              <h4
                class="MuiTypography-root makeStyles-label-1 MuiTypography-h4"
              >
                Contract
              </h4>
            </span>
          </label>
          <label
            class="MuiFormControlLabel-root makeStyles-formLabel-2"
            variant="secondary"
          >
            <span
              aria-disabled="false"
              class="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-5 MuiCheckbox-root MuiCheckbox-colorPrimary MuiIconButton-colorPrimary"
            >
              <span
                class="MuiIconButton-label"
              >
                <input
                  class="PrivateSwitchBase-input-8"
                  data-indeterminate="false"
                  name="Remote"
                  type="checkbox"
                  value=""
                />
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
                  />
                </svg>
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </span>
            <span
              class="MuiTypography-root MuiFormControlLabel-label MuiTypography-body1"
            >
              <h4
                class="MuiTypography-root makeStyles-label-1 MuiTypography-h4"
              >
                Remote
              </h4>
            </span>
          </label>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item"
      >
        <div
          class="MuiFormGroup-root"
        >
          <p
            class="MuiTypography-root makeStyles-textMargin-4 MuiTypography-body1"
          >
            Experience Level
          </p>
          <label
            class="MuiFormControlLabel-root makeStyles-formLabel-2"
            variant="secondary"
          >
            <span
              aria-disabled="false"
              class="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-5 MuiCheckbox-root MuiCheckbox-colorPrimary MuiIconButton-colorPrimary"
            >
              <span
                class="MuiIconButton-label"
              >
                <input
                  class="PrivateSwitchBase-input-8"
                  data-indeterminate="false"
                  name="Fresher"
                  type="checkbox"
                  value=""
                />
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
                  />
                </svg>
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </span>
            <span
              class="MuiTypography-root MuiFormControlLabel-label MuiTypography-body1"
            >
              <h4
                class="MuiTypography-root makeStyles-label-1 MuiTypography-h4"
              >
                Fresher
              </h4>
            </span>
          </label>
          <label
            class="MuiFormControlLabel-root makeStyles-formLabel-2"
            variant="secondary"
          >
            <span
              aria-disabled="false"
              class="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-5 MuiCheckbox-root MuiCheckbox-colorPrimary MuiIconButton-colorPrimary"
            >
              <span
                class="MuiIconButton-label"
              >
                <input
                  class="PrivateSwitchBase-input-8"
                  data-indeterminate="false"
                  name="Mid-Level"
                  type="checkbox"
                  value=""
                />
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
                  />
                </svg>
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </span>
            <span
              class="MuiTypography-root MuiFormControlLabel-label MuiTypography-body1"
            >
              <h4
                class="MuiTypography-root makeStyles-label-1 MuiTypography-h4"
              >
                Mid-Level
              </h4>
            </span>
          </label>
          <label
            class="MuiFormControlLabel-root makeStyles-formLabel-2"
            variant="secondary"
          >
            <span
              aria-disabled="false"
              class="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-5 MuiCheckbox-root MuiCheckbox-colorPrimary MuiIconButton-colorPrimary"
            >
              <span
                class="MuiIconButton-label"
              >
                <input
                  class="PrivateSwitchBase-input-8"
                  data-indeterminate="false"
                  name="Director"
                  type="checkbox"
                  value=""
                />
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
                  />
                </svg>
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </span>
            <span
              class="MuiTypography-root MuiFormControlLabel-label MuiTypography-body1"
            >
              <h4
                class="MuiTypography-root makeStyles-label-1 MuiTypography-h4"
              >
                Director
              </h4>
            </span>
          </label>
          <label
            class="MuiFormControlLabel-root makeStyles-formLabel-2"
            variant="secondary"
          >
            <span
              aria-disabled="false"
              class="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-5 MuiCheckbox-root MuiCheckbox-colorPrimary MuiIconButton-colorPrimary"
            >
              <span
                class="MuiIconButton-label"
              >
                <input
                  class="PrivateSwitchBase-input-8"
                  data-indeterminate="false"
                  name="Executive"
                  type="checkbox"
                  value=""
                />
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
                  />
                </svg>
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </span>
            <span
              class="MuiTypography-root MuiFormControlLabel-label MuiTypography-body1"
            >
              <h4
                class="MuiTypography-root makeStyles-label-1 MuiTypography-h4"
              >
                Executive
              </h4>
            </span>
          </label>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item"
      >
        <div
          class="MuiFormGroup-root"
        >
          <p
            class="MuiTypography-root makeStyles-textMargin-4 MuiTypography-body1"
          >
            Transport
          </p>
          <label
            class="MuiFormControlLabel-root makeStyles-formLabel-2"
            variant="secondary"
          >
            <span
              aria-disabled="false"
              class="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-5 MuiCheckbox-root MuiCheckbox-colorPrimary MuiIconButton-colorPrimary"
            >
              <span
                class="MuiIconButton-label"
              >
                <input
                  class="PrivateSwitchBase-input-8"
                  data-indeterminate="false"
                  name="Metro"
                  type="checkbox"
                  value=""
                />
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
                  />
                </svg>
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </span>
            <span
              class="MuiTypography-root MuiFormControlLabel-label MuiTypography-body1"
            >
              <h4
                class="MuiTypography-root makeStyles-label-1 MuiTypography-h4"
              >
                Metro
              </h4>
            </span>
          </label>
          <label
            class="MuiFormControlLabel-root makeStyles-formLabel-2"
            variant="secondary"
          >
            <span
              aria-disabled="false"
              class="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-5 MuiCheckbox-root MuiCheckbox-colorPrimary MuiIconButton-colorPrimary"
            >
              <span
                class="MuiIconButton-label"
              >
                <input
                  class="PrivateSwitchBase-input-8"
                  data-indeterminate="false"
                  name="Motor - Cycle"
                  type="checkbox"
                  value=""
                />
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
                  />
                </svg>
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </span>
            <span
              class="MuiTypography-root MuiFormControlLabel-label MuiTypography-body1"
            >
              <h4
                class="MuiTypography-root makeStyles-label-1 MuiTypography-h4"
              >
                Motor - Cycle
              </h4>
            </span>
          </label>
          <label
            class="MuiFormControlLabel-root makeStyles-formLabel-2"
            variant="secondary"
          >
            <span
              aria-disabled="false"
              class="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-5 MuiCheckbox-root MuiCheckbox-colorPrimary MuiIconButton-colorPrimary"
            >
              <span
                class="MuiIconButton-label"
              >
                <input
                  class="PrivateSwitchBase-input-8"
                  data-indeterminate="false"
                  name="Bus"
                  type="checkbox"
                  value=""
                />
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
                  />
                </svg>
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </span>
            <span
              class="MuiTypography-root MuiFormControlLabel-label MuiTypography-body1"
            >
              <h4
                class="MuiTypography-root makeStyles-label-1 MuiTypography-h4"
              >
                Bus
              </h4>
            </span>
          </label>
          <label
            class="MuiFormControlLabel-root makeStyles-formLabel-2"
            variant="secondary"
          >
            <span
              aria-disabled="false"
              class="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-5 MuiCheckbox-root MuiCheckbox-colorPrimary MuiIconButton-colorPrimary"
            >
              <span
                class="MuiIconButton-label"
              >
                <input
                  class="PrivateSwitchBase-input-8"
                  data-indeterminate="false"
                  name="Car Pooling"
                  type="checkbox"
                  value=""
                />
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
                  />
                </svg>
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </span>
            <span
              class="MuiTypography-root MuiFormControlLabel-label MuiTypography-body1"
            >
              <h4
                class="MuiTypography-root makeStyles-label-1 MuiTypography-h4"
              >
                Car Pooling
              </h4>
            </span>
          </label>
        </div>
      </div>
    </div>
    <div
      class="MuiGrid-root makeStyles-gridMargin-3 MuiGrid-container MuiGrid-spacing-xs-5 MuiGrid-item MuiGrid-align-items-xs-center MuiGrid-justify-xs-flex-end"
    >
      <div
        class="MuiGrid-root MuiGrid-item"
      >
        <button
          class="MuiButtonBase-root MuiButton-root MuiButton-outlined makeStyles-base-9 makeStyles-small-10 MuiButton-outlinedSecondary"
          tabindex="0"
          type="button"
        >
          <span
            class="MuiButton-label"
          >
            Clear all
          </span>
          <span
            class="MuiTouchRipple-root"
          />
        </button>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item"
      >
        <button
          class="MuiButtonBase-root MuiButton-root MuiButton-contained makeStyles-base-9 makeStyles-small-10 MuiButton-containedPrimary"
          data-testid="ApplyFilter"
          tabindex="0"
          type="button"
        >
          <span
            class="MuiButton-label"
          >
            Apply
          </span>
          <span
            class="MuiTouchRipple-root"
          />
        </button>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
